<?xml version="1.0"?>
<launch>

  <!--PLAY RAWLOG FILE-->
  <include file="$(find mrpt_rawlog)/launch/demo_play_ekf_2d.launch" />

  <!--PARAMETERS-->
  <param name="ellipse_scale" value="1"/>
  <param name="ini_filename" value="$(find mrpt_ekf_slam_2d)/tutorial/kf-slam_demo_2d.ini"/>
  <param name="odom_frame_id" value="/odom"/>
  <param name="global_frame_id" value="/map"/>
  <param name="base_frame_id" value="/base_link"/>
  <!-- for multiple sensors, place comma eg "r1/front_laser/scan,/scan_rear " -->
  <param name="sensor_source" value="/landmark"/>

  <!-- Setting environment variables -->
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find mrpt_ekf_slam_2d)/config/rosconsole.config"/>

  <!--SLAM RUN-->
  <node pkg="rviz" type="rviz" name="rviz_nav" output="screen" args="-d $(find mrpt_ekf_slam_2d)/rviz/  rviz_conf_ekf_2d.rviz"></node>
  <node pkg="mrpt_ekf_slam_2d" type="mrpt_ekf_slam_2d" name="mrpt_ekf_slam_2d" output="screen"></node>
</launch>
